Release/libs/websocketpp/examples/broadcast_server/broadcast_server.cpp

```diff
@@ -19,6 +19,7 @@ using websocketpp::lib::bind;
 
 using websocketpp::lib::thread;
 using websocketpp::lib::mutex;
+using websocketpp::lib::lock_guard;
```

Comment:
This is part of the vendored copy of websocketpp, so we should avoid modifying it.

Reply:
I see, I'll revert changes in this example.